bot/tgbot/middlewares/settings.py

```
@@ -0,0 +1,18 @@
+from typing import Any, Awaitable, Callable, Dict
```

Ruff (0.14.5)

Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)